spec/requests/api/v1/targets/show_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe 'Api::V1::Targets', type: :request do
+  describe 'GET #show' do
+    context 'success' do
+      before { get api_v1_target_url(target.id), as: :json }
+
+      let(:topic) { create(:topic) }
+      let(:target) { create(:target, topic: topic) }
+
+      it 'returns the target as a json' do
+        expect(response.body).to include_json(
```

Comment:
maybe here you could do:
```
expect(json['target']).to include_json(
            id: target.id,
            topic_id: topic.id,
            title: target.title,
            radius: target.radius,
            longitude: target.longitude,
            latitude: target.latitude
)
```

Reply:
i tried but does not work, json['target'] did not catch the json.